admin-dev/themes/new-theme/js/components/module-card.ts

```diff
@@ -423,6 +423,10 @@ export default class ModuleCard {
           mainElement.attr('data-installed', '0');
           mainElement.attr('data-active', '0');
 
+          if (forceDeletion === 'true' || forceDeletion === true && moduleTechName !== 'ps_distributionapiclient') {
+            mainElement.hide();
```

Comment:
I'm not blocking for this, but why not use `.remove()` instead?

Reply:
I agree, remove is better

Reply:
The transition from hide to remove required a small additional tweak. You have to send the event before removing.
(` this.eventEmitter.emit('Module Uninstalled', mainElement);`)

Now it is good :smile: